test/cppsuite/README.md

```diff
@@ -1 +1,148 @@
+| -------- | ----------- |
+| Load     | Initializes the oldest and stable lags using the values retrieved from the configuration file. |
+| Run      | Updates the oldest and stable lags as well as the oldest and stable timestamps. |
+| Finish   | N/A |
+
+## Metrics monitor
+The metrics monitor polls database metrics during and after test execution. If a metric is found to be outside the acceptable range defined in the configuration file then the test will fail. When a test ends, the metrics monitor can export the value of these statistics to a JSON file that can be uploaded to Evergreen or Atlas.
+
+| Phase    | Description |
+| -------- | ----------- |
+| Load     | Retrieves the metrics from the configuration file that need to be monitored during and after the test. |
+| Run      | Checks runtime metrics and fails the test if one is found outside the range defined in the configuration file. |
+| Finish   | Checks post run metrics and fails the test if one is found outside the range defined in the configuration file. Exports any required metrics to a JSON file if the test is successful. |
+
+The components and their implementation can be found in the [component](https://github.com/wiredtiger/wiredtiger/tree/develop/test/cppsuite/src/component) folder.
+
+# Test configuration file
+A test configuration is a text file that is associated with a test and defines the workload. The test configuration file contains top-level settings such as the test duration or cache size and component level settings that define their behavior. The format is the following:
+
+```cpp
+# Top level settings
+duration_seconds=15,
+cache_size_mb=200,
+...,
+# Components settings
+timestamp_manager=
+(
+  enabled=true,
+  oldest_lag=1,
+  stable_lag=1,
+  ...,
+),
+...,
+```
+
+Each test has a default configuration file which is the test name followed by the suffix `_default` and the extension `.txt`. It is possible to use any configuration file when executing a test, see [here](how_to_use_cppsuite.md#running-tests) for more details.
+
+All the different configurable items are defined in [test_data.py](https://github.com/wiredtiger/wiredtiger/blob/develop/dist/test_data.py).
+
+# Logging
+The framework writes traces to stdout using a [logger](https://github.com/wiredtiger/wiredtiger/blob/develop/test/cppsuite/src/common/logger.cpp) and supports the following log levels:
+
+| Logging level | Logging value | Description |
+| ------------- | ------------- | ----------- |
+| 0             | LOG_ERROR     | Message indicated an error faced by the framework or a failure of the test. |
+| 1             | LOG_WARN      | Warning conditions potentially signaling non-imminent errors and behaviors. |
+| 2             | LOG_INFO      | Informational style messages. |
+| 3             | LOG_TRACE     | Low severity messages, useful for debugging purposes. |
+
+A message will not be filtered if the configured logging level is higher or equal to the logging level of the message:
+
```

Comment:
should `ERROR` be `LOG_ERROR` below?

Reply:
Nice catch! Fixed in 1a7399e2ce9547f70f6fceb530fd72de1dbf6660